### PR TITLE
FISH-6276 Stop ArithmeticException being thrown

### PR DESCRIPTION
**Pull request by kalinchan**
# Notes
Unsure how to test this since I couldn't reproduce the issue in the first place.

### CODE REVIEW

**Comment by kalinchan** [comment]
jenkins test please

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HoggingThreadsHealthCheck.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ public void collect(MonitoringWatchCollector collector) {
             .red(1, -2, false, null, null, false);
     }
 
-    private void acceptHoggingThreads(Map<Long, ThreadCpuTimeRecord> recordsById, HoggingThreadConsumer consumer) {
+    private synchronized void acceptHoggingThreads(Map<Long, ThreadCpuTimeRecord> recordsById, HoggingThreadConsumer consumer) {
```
Comment (luiseufrasio) [inline nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HoggingThreadsHealthCheck.java:201]: Why this one was changed to synchronized?
I did not see a relation with the issue.

**Review by luiseufrasio — COMMENTED** [review]
The change will avoid the Exception, but I did not understand the "synchronized" part.

File: nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HoggingThreadsHealthCheck.java (hunk 1 above)
Comment (kalinchan) [reply nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/HoggingThreadsHealthCheck.java:201]: Accident, I will remove.

**Comment by breakponchito** [comment]
Jenkins test please

**Review by luiseufrasio — APPROVED** [review]
LGTM

**Pull request merged**